package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "alinka-website",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```

Comment:
We could set script for running gulp here. We could specify scripts like `"watch": "gulp sass:watch"` etc. It will not require knowledge of npx to run this project